dev/bots/analyze_sample_code.dart

```diff
@@ -746,9 +746,10 @@ dependencies:
     sdk: flutter
   flutter_test:
     sdk: flutter
+  vector_math:
```

Comment:
Can you make this explicit?
```suggestion
  vector_math: any
```

Reply:
Will do!